cmd/mayactl/app/command/volume.go

```diff
@@ -95,6 +95,13 @@ Examples:
 
  # Delete a Volume created in 'test' namespace:
    $ mayactl volume delete --volname <vol> --namespace test
+
+ # Resize a Volume created in default namespace:
+   $ mayactl volume resize --volname <vol> --size <size><unit>
```

Comment:
Can you comment on supported units

Reply:
@mittachaitu let's add one resize test using `mayactl volume resize ` for any Cstor volume we have in travis tests